react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -403,7 +403,7 @@ export const Axes = MinimalTemplate.bind({});
 Axes.args = {
     id: "axes",
     layers: [axes, meshMapLayer, north_arrow_layer],
-    bounds: [432150, 6475800, 439400, 6481500],
+    bounds: [432205, 6475078, 437720, 6481113],
```

Comment:
This bound should be same as axis bounds.
Ideally, project bounds should be calculated based on data bounds of all the layers.

Reply:
We have an issue for automatically calculating the bounds (https://github.com/equinor/webviz-subsurface-components/issues/924), but until we have that in place, then I think using the bounds itself to set the initial viewport is the best solution.